src/components/CardView/CardView.js

```diff
@@ -1,11 +1,17 @@
 import React, { PropTypes } from 'react';
 
 const CardView = (props) => {
-  const { title, text, imageUrl, imageAlt, buttonUrl, buttonText } = props;
+  const { title, text, imageUrl, imageAlt, imageWidth, imageHeight, buttonUrl, buttonText } = props;
 
   let imageEle;
```

Comment:
For the ternary, it's to avoid a really long line. The linter has a line length limit that this breaks unless I split it up. I don't want to use multi-line ternary operators, they're too confusing to follow.

Ele is just short for Element. Tbh, that's just a suffix I've used for years. I think it's a jQuery convention from years ago, so can definitely change! For now, might just make it "image" itself. Was thinking that a suffix to denote JSX elements made sense, but maybe its not needed at all?